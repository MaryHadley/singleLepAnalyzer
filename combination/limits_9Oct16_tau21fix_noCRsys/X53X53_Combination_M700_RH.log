000 (1297361.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297361.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.228:44891>
...
006 (1297361.000.000) 10/10 19:27:26 Image size of job updated: 11336
	2  -  MemoryUsage of job (MB)
	1144  -  ResidentSetSize of job (KB)
...
006 (1297361.000.000) 10/10 19:32:26 Image size of job updated: 283624
	159  -  MemoryUsage of job (MB)
	162348  -  ResidentSetSize of job (KB)
...
006 (1297361.000.000) 10/10 20:17:30 Image size of job updated: 284424
	160  -  MemoryUsage of job (MB)
	163232  -  ResidentSetSize of job (KB)
...
005 (1297361.000.000) 10/10 20:21:49 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:53:59, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:53:59, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1629  -  Run Bytes Sent By Job
	322  -  Run Bytes Received By Job
	1629  -  Total Bytes Sent By Job
	322  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1097        1   3824814
	   Memory (MB)          :      160     3072      3072
...
