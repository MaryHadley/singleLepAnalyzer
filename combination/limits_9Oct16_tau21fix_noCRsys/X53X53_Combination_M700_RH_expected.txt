# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0    0.10527   0.053201    0.22106  0.073215    0.15742
